src/styles/stickyNavBar.css

```diff
@@ -1,34 +1,134 @@
 .navbar-wrapper {
-    margin-top: 5px; 
+  width: 100%;
+  overflow-x: hidden;
+  position: relative;
+  z-index: 1000;
+  transition: background-color 0.1s ease-out; /* Quick transition for visual changes */
+}
+
+.navbar-wrapper.fixed-top {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 1000;
+}
+
+.navbar {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 20px;
+  padding-top: 20px; 
+  padding-bottom: 30px; 
+  gap: 10px;
+  font-family: 'Inter', sans-serif;
+  background-image: url("../../public/built-background.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-position: center;
+  color: black;
+  width: 100%;
+}
+
+.button {
+  background-color: #DB9066;
+  border: none;
+  color: black;
+  padding: 10px 20px;
+  font-size: 14px; 
+  border-radius: 30px; 
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  border-radius: 50px;
+  font-family: 'Inter', sans-serif;
+  white-space: nowrap;
+  box-sizing: border-box;
+}
+
+.button:hover {
+  color: #000000;
+  background-color: rgb(243, 221, 205);
+}
+
+.hamburger-menu {
+  display: none; 
+  flex-direction: column;
+  gap: 5px;
+  background: none;
+  border: none;
+  cursor: pointer;
+  z-index: 1100;
+  position: absolute;
+  top: 20px;
+  right: 20px; 
+}
+
+.hamburger-menu .line {
+  width: 30px;
+  height: 3px;
+  background-color: #62250C;
+  border-radius: 2px;
+}
+
+.nav-links {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+}
+
+.nav-links.show {
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  top: 0;
+  right: 0; 
+  background-color: rgba(243, 221, 205, 0.9);
+  width: 50%;
+  height: 100%;
+  padding: 10%;
+  z-index: 1101;
+  transform: translateX(100%); 
+  animation: slide-in-right 0.3s forwards;
+}
+
+@keyframes slide-in-right {
+  to {
+    transform: translateX(0); 
+  }
+}
+
+@media (max-width: 500px) {
```

Comment:
Some of the text appears to off center when viewed on a smaller phone such as the iPhone SE, after the hamburger menu is opened.